build.gradle

```diff
@@ -63,14 +63,16 @@ subprojects {
 
     repositories {
         jcenter()
+        // Spring milestone repo for spring-cloud release candidates
+        maven { url 'https://repo.spring.io/milestone/' }
     }
 
     ext {
-        springBootVersion = '1.5.19.RELEASE'
-        springCloudVersion = 'Edgware.SR2'
+        springBootVersion = '2.2.1.RELEASE'
+        springCloudVersion = 'Hoxton.RC2'
```

Comment:
I was able to run the app locally (on the demo app). Does not require spring cloud on the classpath. The spring cloud version does get included in the library pom file, hence to use it the Spring's milesonte repo has to be included. I would not spend much time on it now, since the next version will be released when the moment Spring Cloud is released as well